### ISSUE TITLE
initial fetch class stuck in angularJs project

### ISSUE THREAD

**Issue by web-dave** [Zignd/HTML-CSS-Class-Completion#15]
My css file is (build with scss) 7563 lines..
When i open a html template the first time  vscode stuck when fetching css classe.
I musst uninstall it to be able to do my work. :(.
Any hints?

**Comment by Zignd** [comment]
Would you mind testing it again? I completely rewrote the extension and the newest release is already on the Marketplace. By the way, thanks for your report and sorry for taking so long to respond.

**Comment by web-dave** [comment]
Nothing chnaged.
I hit the cache btn but nothing happens then vscode stuck.

**Comment by Zignd** [comment]
Hi there, sorry asking for it again, but would you mind testing it on more time, please? I released a new version to fix a problem I managed to identify related to parsing documents in parallel and asynchronously.

**Comment by michael-woodruff** [comment]
I'm seeing the same issue. Angular project with scss, vscode will completely lock up. I left it running for 30 min and it never finished whatever it was doing.

**Comment by maganuk** [comment]
Im having the same issue

**Comment by Zignd** [comment]
Closing because it seems to be a duplicate of #12.
